src/main/java/kostka/moviecatalog/dto/RuntimeConfigDto.java

```diff
@@ -0,0 +1,34 @@
+package kostka.moviecatalog.dto;
+
+import org.hibernate.validator.constraints.Range;
+
+import javax.validation.constraints.NotNull;
+//TODO: instead of config name user config enums, limit could be string as it comes from FE
+public final class RuntimeConfigDto {
```

Comment:
This is Dto for all possible runtime configs or only for the MoviesLimitRuntimeConfig? Because if ti is a general one that the field `limit` doesn't make sense...